src/docs/extract.ml

```diff
@@ -12,6 +12,7 @@ and declaration_doc =
   | Value of value_doc
   | Type of type_doc
   | Class of class_doc
+  | Module of module_doc
```

Comment:
You could consider adding a `sort : Type.obj_sort` (object|module|actor) and then using the sort field of `ObjBlockE` below to populate it. Then you could emit doc for objects and actors too.

Reply:
Gotcha; thanks. I was 50/50 on generalizing this for the initial PR, but I went ahead and set everything up (along with a few improvements for module and actor classes).